src/components/Map.jsx

```diff
@@ -1,55 +1,68 @@
+import { useEffect, useState } from 'react'
+import axios from 'axios'
 import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
 import classes from "./Map.module.scss";
-
-const DUMMY_COORDINATES = [
-  {
-    id: "i1",
-    name: "bills",
-    coords: {
-      lat: 35.668884986902434,
-      long: 139.70603408498235,
-    },
-  },
-  {
-    id: "i2",
-    name: "Harry's Sandwich Company",
-    coords: {
-      lat: 35.66702189118264,
-      long: 139.70644521236878,
-    },
-  },
-  {
-    id: "i3",
-    name: "SOUP STOCK TOKYO CAFE",
-    coords: {
-      lat: 35.66597014771571,
-      long: 139.711036633326,
-    },
-  },
-  {
-    id: "i4",
-    name: "Fratelli Paradiso",
-    coords: {
-      lat: 35.6674098641632,
-      long: 139.70841296420758,
-    },
-  },
-];
-
-const DUMMY_LOCATION = { lat: 35.66756203584793, long: 139.7077023534186 };
+import Loading from "./Loading";
 
 const Map = () => {
-  const { lat, long } = DUMMY_LOCATION;
-  const coords = [lat, long];
+  const [data, setData] = useState([])
+  const [currentLocation, setCurrentLocation] = useState('')
+  const [loading, setLodaing] = useState(true)
+  const [coords, setCoords] = useState([])
   const mapZoomLevel = 15;
 
-  const markers = DUMMY_COORDINATES.map((el) => {
-    const { lat, long } = el.coords;
-    const coords = [lat, long];
+  useEffect(() => {
+    if (!loading) {
+      const BASEURL = 'https://api.foursquare.com/v3/places/search?'
+      const parameters = {
+        client_id: import.meta.env.VITE_FORESQUARE_CLIENT_ID,
```

Comment:
I think these are not required (Only required in the legacy api)
![image](https://user-images.githubusercontent.com/6191116/227774243-1d6af691-bd5b-4164-8e07-9ea18ed2ded1.png)